src/components/general/Dropdown/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useRef, useCallback, FC, useEffect } from 'react';
+import {
+    TextInput,
+    DropdownArrow,
+    Menu,
+    useClickOutsideOverlayPortal,
+    RelativeOverlayPortal,
+} from '@equinor/fusion-components';
+import styles from './styles.less';
+
+type DropdownOption = {
```

Comment:
Perhaps the `DropdownOption` type should have `selected?: boolean;` property to indicate which option is currently selected?